src/NzbDrone.Core/Indexers/BitMeTv/BitMeTvSettings.cs

```diff
@@ -47,6 +54,13 @@ public BitMeTvSettings()
         [FieldDefinition(4, Type = FieldType.Textbox, Label = "Minimum Seeders", HelpText = "Minimum number of seeders required.", Advanced = true)]
         public int MinimumSeeders { get; set; }
 
+        [FieldDefinition(5, Type = FieldType.Textbox, Label = "Seed Ratio", HelpText = "The ratio a torrent should reach before stopping, empty is download client's default")]
+        public string UiSeedRatio
```

Comment:
I didn't comment on this earlier because there were bigger fish to fry. But why this workaround in every settings class? Shouldn't it simply be a double? The value conversion should be handled in the UI or the SchemaBuilder.